tests/ci_folder_parsing.py

```diff
@@ -385,4 +334,280 @@ def test_empty(self, steps, local_class, operating_system, token=None):
 
 
 if __name__ == "__main__":
-    aetest.main()
+
+    # Create the parser
+    my_parser = argparse.ArgumentParser(description="Optional arguments for 'nose'-like tests")
+
+    my_parser.add_argument('-o', "--operating_system",
+                        type=str,
+                        help='The OS you wish to filter on',
+                        default=None)
+    my_parser.add_argument('-c', "--class_name",
+                        type=str,
+                        help="The Class you wish to filter on, (not the Test File)",
+                        default=None)
+    my_parser.add_argument('-t', "--token",
+                        type=str,
+                        help="The Token associated with the class, such as 'asr1k'",
+                        default=None)
+    my_parser.add_argument('-f', "--display_only_failed",
+                        help="Displaying only failed classes",
+                        action='store_true')
+    my_parser.add_argument('-n', "--number",
+                        type=int,
+                        help="The specific unittest we want to run, such as '25'",
+                        default=None)
+    args = my_parser.parse_args()
+
+    _os = args.operating_system
+    _class = args.class_name
+    _token = args.token
+    _display_only_failed = args.display_only_failed
+    _number = args.number
+
+    if _number and (not _class or not _number):
+        sys.exit("Unittest number provided but missing supporting arguments:"
+                "\n* '-c' or '--class_name' for the parser class"
+                "\n* '-o' or '--operating_system' for operating system")
+
+    # This is the list of Classes that currently have no testing. It was found during the process
+    # of converting to folder based testing strategy
+    CLASS_SKIP = {
```

Comment:
What if we were to put this information into a separate yaml file instead of keeping it as a hard coded dictionary? Would make it easier to and and remove entries as unittests are developed I'd think. Same think with `EMPTY_SKIP`

Reply:
That makes sense, either yaml or txt, I'll add that

Reply:
it would be good to verify this list again, I think some were pending a PR as an example, and there may be tests already.